server/shared/configFetcher.go

```diff
@@ -60,6 +63,22 @@ func (cf *configFetcher) fetch(c context.Context, timeout time.Duration, config
 	ctx, cancel := context.WithTimeout(c, timeout)
 	defer cancel()
 
+	if cf.configAPI.IsConfigured() {
+		log.Trace().Msgf("ConfigFetch[%s]: getting config from external http service '%s'", cf.repo.FullName)
```

Comment:
since drone and woodpecker pipeline config differ anyway ...
I think we can add multi pipeline support right away & we should also send the origin config from repo (if exist) to that endpoint

Reply:
Could not agree more :-)